index.md

```diff
@@ -13,21 +16,34 @@ electronic structure codes, it has become possible to perform large sets
 of calculations automatically. This is the burgeoning area of
 high-throughput ab initio computation. Such calculations have been used
 to create large databases containing the calculated properties of
-existing and hypothetical materials, many of which have appeared online:
-
-- [the AFLOW distributed materials property repository](http://aflowlib.org/)
-- [the Harvard Clean Energy Project Database](http://molecularspace.org/)
-- [the Materials Cloud](http://materialscloud.org/)
-- [the Materials Project](http://materialsproject.org/)
-- [the NoMaD (Novel Materials Discovery) Repository](http://nomad-repository.eu/)
-- [the Open Quantum Materials Database](http://oqmd.org/)
-- [the Computational Materials Repository](http://cmr.fysik.dtu.dk/)
-- [the Data Catalyst Genome](http://suncat.stanford.edu/)
-- [the Materials Platform for Data Science](http://mpds.io/)
-- [the Joint Automated Repository for Various Integrated Simulations](https://jarvis.nist.gov)
-- ...
+existing and hypothetical materials, many of which have appeared online.
+
+We have released version 1.0 of the OPTIMADE specification, with several
+databases already providing implementations. A full list is available on
+the [OPTIMADE providers dashboard](https://www.optimade.org/providers-dashboard/).
+
+## How to cite OPTIMADE
 
 Should you wish to cite the OPTIMADE specification, please use the following:
 
 - Andersen *et al*, OPTIMADE: an API for exchanging materials data (2021) [arXiv:2103.02068](https://arxiv.org/abs/2103.02068){:target=_blank}
-- Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195050](https://doi.org/10.5281/zenodo.4195050){:target=blank}
+- Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195050](https://doi.org/10.5281/zenodo.4195050){:target=_blank}
+
+## Get involved
+
+All of our work is openly available under the
+[Materials-Consortia](https://github.com/Materials-Consortia/) organization on
+GitHub and welcome all contributions.
+
+We would love to help you create and register your own OPTIMADE API
+implementations and hear your feedback on the specification.
+We meet monthly in the "OPTIMADE" room on Jitsi; everyone is welcome to
+join the discussion, please feel free to reach out on the mailing list
+`dev[at]optimade.org` to register your interest.
+The next meeting will take place on
+[Friday April 23rd at 15:00 UTC](https://www.worldtimebuddy.com/?pl=1&lid=2643743,5128581,5391959,2950159&h=2643743&date=4/23/2021%7C3&hf=1){:target=_blank}.
+
+With the support of [CECAM](www.cecam.org){:target=_blank}, we hold annual
+workshops to discuss and develop the specification and related ideas.
+The 2021 workshop will be held virtually and the announcement will appear
+on the CECAM website soon.
```

Comment:
This should be highlighted, especially because this is something the easily gets forgotten when updating the webpage. If it's highlighted, there might be a higher chance of us noticing it needs to be updated :)

Reply:
I'm happy to just be "in charge" of this, as we will also have to update the monthly meeting dates as they are organized. I will raise an issue if this gets merged. I think it might be preferable in this case to avoid PR spam and permit a push to master directly for commits that just change this rolling date. Probably a bit overengineered, but we could also have a `next_meeting` file that does not automatically trigger code reviews, but this might require some jekyll wizardry...

Reply:
I'll dump all responsibilities regarding this point on you then 😅

Reply:
The 2021 workshop is already available on the CECM WEBSITE: https://www.cecam.org/workshop-details/52
It might change though... because it is going to be online. We have a meeting with CECAM on the 7th of April

Reply:
I've just switched to the `include` approach

Reply:
Wrongly resolved this conversation.
The tagged line can/should be updated according to @gmrigna's suggestion above.

Reply:
I don't think so, I added a link to the search that includes the full list of OPTIMADE events, as it sounds like a new event will be made for 2021?

Reply:
Not if the dates are still subject to change; of course we can add it as soon as it is finalized (sounds like that will be a week or so)

Reply:
The dates are fixed... We are just going to discuss about the tools (Zoom, Teams, GatherTown, ...) that we are going to use...  I am just not sure whether they are going to create a new page for the virtual event and keep the current one for the workshop of next year, or if they will use the current page. So, it is indeed better to wait our meeting on the 7th of April...